dpdata.py

```diff
@@ -159,6 +160,12 @@ def __fixup_nomail_flag(self):
                 # Empty out NOMAIL_REASON if NOMAIL not set
                 dp_donorrecord['NOMAIL_REASON'] = ''
 
+    def __copy_spouse_email(self):
```

Comment:
Okay, let's move forward with this one for now. I suspect we are introducing more complexity by spreading out the logic, but there may not be an easy fix here.